CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ NEW FEATURES:
 (#1206)
 * `dep ensure -no-vendor -dry-run` now exits with an error when changes would
 have to be made to `Gopkg.lock`. This is useful for CI. (#1256)
+* Allow override of cache directory location using environment variable
+`DEPCACHEDIR`. (#1234)
```

Comment:
We just released v0.3.2, so please move this under v0.3.3. We have started using release milestones to avoid this in future.
Thanks!

Reply:
Done with rebase.